ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -113,6 +165,71 @@ const triggerSet: OopsyTriggerSet<Data> = {
           delete data.raiseTracker[matches.targetId];
       },
     },
+    {
+      id: 'General Overwritten Mit',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: targetMitAbilityIds.concat(partyMitAbilityIds) }),
+      mistake: (data, matches) => {
+        const isTargetMit = targetMitAbilityIds.includes(matches.id);
+        const isPartyMit = partyMitAbilityIds.includes(matches.id);
+        if (isTargetMit && matches.targetId === 'E0000000') // missed
+          return;
+        if (isPartyMit && !data.party.partyIds_.includes(matches.sourceId))
+          return;
+        if (isPartyMit && matches.targetId !== matches.sourceId)
+          return;
+
+        const mitTracker = isTargetMit
+          ? (data.targetMitTracker ??= {})[matches.targetId]
+          : data.partyMitTracker;
+        const newTime = new Date(matches.timestamp).getTime();
+        const newSource = data.ShortName(matches.source);
+        const lastTime = mitTracker?.[matches.id]?.time;
+        const lastSource = mitTracker?.[matches.id]?.source;
+
+        if (isTargetMit) {
+          ((data.targetMitTracker ??= {})[matches.targetId] ??= {})[matches.id] = {
+            time: newTime,
+            source: newSource,
+          };
+        }
+        if (isPartyMit) {
+          (data.partyMitTracker ??= {})[matches.id] = {
+            time: newTime,
+            source: newSource,
+          };
+        }
```

Comment:
```suggestion
        const mitTracker = isTargetMit
          ? ((data.targetMitTracker ??= {})[matches.targetId] ??= {})
          : (data.partyMitTracker ??= {});
        const newTime = new Date(matches.timestamp).getTime();
        const newSource = data.ShortName(matches.source);
        const lastTime = mitTracker[matches.id]?.time;
        const lastSource = mitTracker[matches.id]?.source;

        mitTracker[matches.id] = {
          time: newTime,
          source: newSource,
        };
```

one last simplification